go/vt/vttablet/tabletserver/vstreamer/rowstreamer.go

```diff
@@ -297,6 +310,8 @@ func (rs *rowStreamer) streamQuery(conn *snapshotConn, send func(*binlogdatapb.V
 		if err != nil {
 			return err
 		}
```

Comment:
i think you want to wrap this in error checking since otherwise you'll just ignore it here